main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -1223,6 +1223,16 @@ object ParsedAst {
       */
     case class Debug(sp1: SourcePosition, kind: ParsedAst.DebugKind, exp: ParsedAst.Expression, sp2: SourcePosition) extends ParsedAst.Expression
 
+    /**
+      * Instanceof expression (currently a binary op - maybe change the syntax?).
+      *
+      * @param exp the expression.
+      * @param name the Java class or interface name.
+      * @param sp2  the position of the last character in the expression.
+      */
+    case class Instanceof(exp: ParsedAst.Expression, name: Name.JavaName, sp2: SourcePosition) extends ParsedAst.Expression
```

Comment:
I would like to ask for one annoying change:

If we could keep similar AST nodes grouped together; so I think I would like `Instanceof` (named `InstanceOf`) to appear directly after Ascribe and directly before CheckedCast. This goes both for the ASTs and for all pattern matches.

Sorry! But it helps keep things neat :)

Reply:
All constructors and match cases should be reordered.